README.md

```diff
@@ -109,6 +109,15 @@ SELECT * FROM TABLE(storage.system.list('/tmp/trino-storage'));
  2023-05-03 12:14:22.107 UTC |   12 | /tmp/trino-storage/test.txt
 ```
 
+```sql
+SELECT * FROM TABLE(storage.system.cat('csv', '/tmp/trino-storage/test.txt'));
```

Comment:
`cat` is not very obvious, how about `read_file`? If this only works with local files, not remote URLs, maybe `read_local_file`.

Reply:
`read_file` looks better. This table function supports accessing files on S3, so I would avoid including "local". 
```
trino> SELECT * FROM TABLE(storage.system.cat('csv', 's3a://trino-ci-test/test/test.txt'));
     col
-------------
 hello world
```